dotcom-rendering/src/web/components/FixedSmallSlowVMPU.tsx

```diff
@@ -19,6 +19,8 @@ export const FixedSmallSlowVMPU = ({
 	showAge,
 	index,
 }: Props) => {
+	if (!trails[0]) return null;
```

Comment:
Tiny though - Would
```
if (trails.length === 0) return null
``` 

Be a more clear check in terms of readability and what we're trying to express? It feels like searching for the `0` index is 1 step inferred to "if the list is empty".

Reply:
Good shout!